### ISSUE TITLE
Error running tests

### ISSUE THREAD

**Issue by ulrichSchreiner** [kellyjonbrazil/jc#227]
hi, 

when i clone the repository and do a simple
~~~
$ python setup.py test
~~~

i get a lot of errors:

~~~

======================================================================
ERROR: test_foo_centos_7_7 (tests._test_foo.MyTests)
Test 'foo' on Centos 7.7
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/usc/tmp/jc/tests/_test_foo.py", line 13, in setUp
    with open(os.path.join(THIS_DIR, os.pardir, 'tests/fixtures/centos-7.7/foo.out'), 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/usc/tmp/jc/tests/../tests/fixtures/centos-7.7/foo.out'

======================================================================
ERROR: test_foo_nodata (tests._test_foo.MyTests)
Test 'foo' with no data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/usc/tmp/jc/tests/_test_foo.py", line 13, in setUp
    with open(os.path.join(THIS_DIR, os.pardir, 'tests/fixtures/centos-7.7/foo.out'), 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/usc/tmp/jc/tests/../tests/fixtures/centos-7.7/foo.out'

======================================================================
ERROR: test_foo_s_centos_7_7 (tests._test_foo_s.MyTests)
Test 'foo' on Centos 7.7
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/usc/tmp/jc/tests/_test_foo_s.py", line 17, in setUp
    with open(os.path.join(THIS_DIR, os.pardir, 'tests/fixtures/centos-7.7/foo.out'), 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/usc/tmp/jc/tests/../tests/fixtures/centos-7.7/foo.out'

======================================================================
ERROR: test_foo_s_nodata (tests._test_foo_s.MyTests)
Test 'foo' with no data
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/usc/tmp/jc/tests/_test_foo_s.py", line 17, in setUp
    with open(os.path.join(THIS_DIR, os.pardir, 'tests/fixtures/centos-7.7/foo.out'), 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/usc/tmp/jc/tests/../tests/fixtures/centos-7.7/foo.out'

======================================================================
FAIL: test_stat_freebsd12 (tests.test_stat.MyTests)
Test 'stat /foo/*' on FreeBSD12
~~~

the ARCH package is not installable at the moment because of these errors.

**Comment by kellyjonbrazil** [comment]
Hi - thanks for reporting this!

I had not tried running tests in that manner before (`python setup.py test`) so I have never noticed it. I am able to replicate this on my machine.

It looks like the only tests that are failing are the template test files (`_test_foo.py` and `_test_foo_s.py`)

I suppose there are several different options:
- Move these template files somewhere else so `setup.py` doesn't find them (or maybe rename them)
- Modify the template files so they noop (comment nearly everything out)
- Modify `setup.py` in some way so it behaves the same as `python3 -m unittest`

Let me see what makes sense here.

**Comment by kellyjonbrazil** [comment]
This is now fixed in the `dev` branch and the fix will be included in the v1.18.7 release.

I ended up simply moving the template test files into a `templates` directory under `tests`. As a quick fix you can also just delete `tests/_test_foo.py` and `tests/_test_foo_s.py` for now.

**Comment by kellyjonbrazil** [comment]
I'm going to release this now in the v1.18.7 release on PIP-only. If things test ok, then I'll bump the github release as well. Thanks!
